erizo/src/erizo/DtlsTransport.cpp

```diff
@@ -106,7 +107,11 @@ DtlsTransport::DtlsTransport(MediaType med, const std::string &transport_name, c
     iceConfig_.ice_components = comps;
     iceConfig_.username = username;
     iceConfig_.password = password;
-    ice_ = NicerConnection::create(io_worker_, iceConfig_);
+    if (iceConfig_.use_nicer) {
+      ice_ = NicerConnection::create(io_worker_, iceConfig_);
```

Comment:
Good point. I'm actually thinking of changing them so both use `=` instead of reset. The point is that I prefer the `create` method to return a shared_ptr instead of a raw pointer.